code/modules/cargo/packs.dm

```diff
@@ -1324,7 +1376,8 @@
 					/obj/item/reagent_containers/glass/rag,
 					/obj/item/grenade/chem_grenade/cleaner,
 					/obj/item/grenade/chem_grenade/cleaner,
-					/obj/item/grenade/chem_grenade/cleaner)
+					/obj/item/grenade/chem_grenade/cleaner,
+					/obj/item/soap/deluxe)
```

Comment:
Why tho. The entire point of the deluxe soap is that it's captain-exclusive

Reply:
There are many ways to get the soap, meta station has some in maints, you can get it form space, and some maps have them in bathrooms I think